Libplanet.Net/Consensus/Context.Async.cs

```diff
@@ -11,15 +11,11 @@ public partial class Context
         /// <summary>
         /// Starts round #0 of consensus for <see cref="Height"/>.
         /// </summary>
-        /// <param name="lastCommit">A <see cref="Block.LastCommit"/> from previous block.
-        /// </param>
-        public void Start(BlockCommit? lastCommit = null)
+        public void Start()
```

Comment:
With this scheme, `Context.Start()` seems to be less meaningful than before, though still we have advantages separating constructor from this for tests.

Reply:
That's one of the reasons. Aside from testing, we might also want to control when a `Context` "starts" from a node. It might not be necessary, but as we will lose more control, it should then be clearly documented **when** a `Context` should be made. 🙄

Reply:
Agree